packages/flutter_tools/lib/src/base/config.dart

```diff
@@ -53,8 +62,10 @@ class Config {
     }
   }
 
+  /// The default directory name for Flutter's configs
```

Comment:
Please add some more context on how this is used. For example, talk about how if the config is written in home it is appended to the name, and if xdg then not.